soccer/gameplay/skills/capture.py

```diff
@@ -23,7 +23,10 @@ class State(Enum):
         course_approach = 1
         fine_approach = 2
 
-    def __init__(self):
+    ## Capture Constructor
+    # onlyApproach - If true, any turning functions are turned off,
+    # useful for using capture to reflect/bounce moving ballls.
+    def __init__(self, onlyApproach=False):
```

Comment:
Maybe rename `onlyApproach` to `faceBall` or something similar for clarity.  The name right now doesn't really convey that it affects facing and nothing else.